pkg/controller/plan/kubevirt.go

```diff
@@ -698,10 +698,14 @@ func (r *KubeVirt) setKvmOnPodSpec(podSpec *core.PodSpec) {
 		}
 		podSpec.NodeSelector["kubevirt.io/schedulable"] = "true"
 		container := &podSpec.Containers[0]
+		// Ensure that the pod is deployed on a node where /dev/kvm is present.
```

Comment:
the previous location of this comment was better - it is achieved by setting the 'requests'